lib/modules/manager/pub/artifacts.spec.ts

```diff
@@ -0,0 +1,230 @@
+import { join } from 'upath';
+import { envMock, mockExecAll } from '../../../../test/exec-util';
+import { env, fs } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import type { UpdateArtifactsConfig } from '../types';
+import * as pub from './artifacts';
```

Comment:
```suggestion
import * as pub from '.';
```
required for branch coverage , also please import function only instead of star import.